apps/b2b/components/Input.tsx

```diff
@@ -0,0 +1,43 @@
+import classNames from 'classnames'
+import {
+  ClassAttributes,
+  ForwardedRef,
+  forwardRef,
+  HTMLAttributes,
+  HTMLProps,
+} from 'react'
+
+interface InputProps
+  extends HTMLAttributes<HTMLInputElement>,
+    ClassAttributes<HTMLInputElement>,
+    HTMLProps<HTMLInputElement> {
+  type?:
+    | 'text'
+    | 'email'
+    | 'number'
+    | 'password'
+    | 'tel'
+    | 'file'
+    | 'date'
+    | 'search'
```

Comment:
i don't think you need to redeclare this. just extend the proper input interface from react.